content/guides/installation.md

```diff
@@ -15,6 +15,8 @@ The next section tells you how to install webpack locally in a project.
 
 ### Local Installation
 
+The latest released webpack version is: [![GitHub release](https://img.shields.io/github/release/webpack/webpack.svg?style=flat-square)](https://github.com/webpack/webpack/releases)
```

Comment:
Maybe `The latest webpack release is: ...` would sound slightly better. Feel free to ignore though, somewhat nitpicky.